libs/commands/publish/src/index.ts

```diff
@@ -973,14 +977,25 @@ class PublishCommand extends Command {
     };
     process.on("log", logListener);
 
+    let queue;
+    if (this.options.throttle) {
+      queue = new TailHeadQueue(
+        this.options.throttleSize !== undefined ? this.options.throttleSize : 25,
+        (this.options.throttleDelay !== undefined ? this.options.throttleDelay : 30) * 1000
+      );
+    } else {
+      queue = new ImmediateQueue();
+    }
     const mapper = pPipe(
       ...[
         (pkg: Package) => {
           const preDistTag = this.getPreDistTag(pkg);
           const tag = !this.options.tempTag && preDistTag ? preDistTag : opts.tag;
           const pkgOpts = Object.assign({}, opts, { tag });
 
-          return pulseTillDone(npmPublish(pkg, pkg.packed.tarFilePath, pkgOpts, this.otpCache))
+          return pulseTillDone(
```

Comment:
See previous comment, let's just use the old logic if throttle is not enabled

Reply:
How do you envision this change ? A ternary that would look like `has_queue ? queue.queue(() => npmPublish(pkg, pkg.packed.tarFilePath, pkgOpts, this.otpCache)) : npmPublish(pkg, pkg.packed.tarFilePath, pkgOpts, this.otpCache)` ?

Reply:
Yeah, exactly, thank you